### ISSUE TITLE
Tensor Flow work log

### ISSUE THREAD

**Issue by liux0229** [liux0229/scratch#2]
https://github.com/tensorflow/models/blob/master/official/mnist/mnist.py

Partial network

INFO:tensorflow:global step 300: loss = 0.6850 (6.183 sec/step)

eval/Recall_5[1]eval/Accuracy[0.7775]

Full network

INFO:tensorflow:global step 300: loss = 0.2631 (8.409 sec/step)

eval/Accuracy[0.8725]eval/Recall_5[1]

**Comment by liux0229** [comment]
- Why does small batch learning rate curve (MSE) is so spiky?
- The gradient of an operator (its weight and its x) depends on the gradient of the operator that depends on it (their x), as well as its own input (x)
